sbin/build.sh

```diff
@@ -826,64 +829,54 @@ generateSBoM() {
     do
       classpath+=$(cygpath -w "${jarfile}")";"
     done
-    sbomJson=$(cygpath -w "${sbomJson}")
-    javaHome=$(cygpath -w "${javaHome}")
   fi
 
+  echo "${classpath}"
+}
+
+# Generate the SBoM
+generateSBoM() {
+  if [[ "${BUILD_CONFIG[CREATE_SBOM]}" == "false" ]] || [[ ! -d "${CYCLONEDB_DIR}" ]]; then
+    echo "Skip generating SBOM"
+    return
+  fi
+
+  local javaHome="$(setupAntEnv)"
+
+  buildCyclonedxLib "${javaHome}"
+  # classpath to run java app TemurinGenSBOM
+  local classpath="$(getCyclonedxClasspath)"
+
+  local sbomTargetName=$(getTargetFileNameForComponent "sbom")
+  # Remove the tarball / zip extension from the name to be used for the SBOM
+  if [[ "$OSTYPE" == "cygwin" ]] || [[ "$OSTYPE" == "msys" ]]; then
```

Comment:
We could simplify this and not require the `if` clause by just stripping both e.g. `sbomTargetName=$(echo "${sbomTargetName}.json" | sed -e "s/\.zip//" -e "s/\.tar\.gz//")
But I'm ok either way.